source/fb_nextgen_mmm_v20.0.func.R

```diff
@@ -774,9 +789,15 @@ f.mmm <- function(...
       #####################################
       #### Get hyperparameter sample
       
+      # Let us create a vector of hyperparameters using Nevergrad.
+      nevergrad_hp <- optimizer$ask()
+      # Now we must cast it to the bounds.
+      # ... TODO ! This is missing
+
       hypParamSam <-  sapply(hyperParams, function(x) {if (length(x) > 1) { x[i] } else {x} }); hypParamSam
```

Comment:
hypParamSam needs to be generated from your 1+1 vector nevergrad_hp instead of hyperParams from my lhs. Also the next row: hypParamSamName needs to have the same hypPar names as your nevergrad_hp. Then we're good to go:)

Reply:
At line 853 below we do fill hypParamSam with nevergrad_hp, right ? So I don't get it.
I'm new to the R syntax so I might be wrong :-)

Reply:
line 853? I can't see it somehow :)

Reply:
From the oral conversation today I guess the code is actually ok.
Next versions:
- including two-objective optimization
- including constraints
- parallelizing, maybe